src/whylogs/whylabs_client/wrapper.py

```diff
@@ -37,30 +57,56 @@ def start_session() -> None:
 
 
 def upload_profile(dataset_profile: DatasetProfile) -> None:
-    dataset_timestamp = None
+    client = _get_whylabs_client()
```

Comment:
since `client` is only used in `_upload_guest_session` here, might as well generate it within that method, rather than on this line

Reply:
Fixed